cloud-service-providers/azure/promptflow/data/manual_info/manuals/product_info_10.md

```diff
@@ -0,0 +1,189 @@
+# Information about product item_number: 10
```

Comment:
Where did this .md/data come from? Do we have the rights to publish this?

Reply:
I am using Azure samples github page demo (https://github.com/Azure-Samples/contoso-chat) and just build on top of that.

Reply:
Okay.  A little worried that copying this data as-is into a different repo with the License file may be a violation (https://github.com/Azure-Samples/contoso-chat/blob/main/LICENSE).

Would it be possible for us to download the data from the Azure owned repo instead of copying it into this GitHub?

Reply:
One thing I can do is that I can modify the README and point that the first step is to clone the Azure-samples repo (https://github.com/Azure-Samples/contoso-chat/) and then make folder where tweaks need to be made. Would that work?

Reply:
Yes, that would be much better. I know it's an extra step, but it would make the legal aspect a lot cleaner if the end-user is downloading.

Reply:
ok, modified it and only kept the original files. Should be good now. Thanks